src/js/components/Header/propTypes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    sticky: PropTypes.oneOf(['scrollup']),
```

Comment:
Saw the earlier thread about whether a boolean should be included. Fine with this direction based on the discussion.

Reply:
yes we discussed that if we want to enhance this more we will add a boolean but for now we are going to just have the `scrollup`